packages/admin/src/mui/menu/Context.tsx

```diff
@@ -15,7 +15,7 @@ export const MenuContext = React.createContext<IMenuContext>({
     toggleOpen: () => {
         // nothing
     },
-    headerHeight: 0,
+    headerHeight: 0, //TODO: should we move the headerHeight to another context? Makes this sense in Menu? -> Maybe MasterLayoutContext?
```

Comment:
Yes, I think we should make a `MasterLayoutContext` for this.

Reply:
@nsams whats do you think? If so, i will remove that in a separat MR because i think it is out of scope from this Mr.

Reply:
Why is it in context anyway? Can't it get passed as prop? Or could it be part of the theme? (it won't change, right?)